base_app/src/app/app.component.ts

```diff
@@ -615,9 +617,10 @@ export class AppComponent implements OnInit, AfterViewInit /*, OnChanges */ {
         try {
             if ((cycle * this._refreshRate) > this._milliSecondsBeforeAutoPageReload) {
                 sessionStorage.autoReload = 'true';
-                setTimeout(() => {
-                    document.location.reload();
-                }, 1);
+                //setTimeout(() => {
```

Comment:
Won't this break the automatic reload functionality. I do want to get it out of the system eventually, but I don't think we can do that until we've proven all memory leaks are resolved.

Reply:
Good catch, this was left over from some testing and should definitely not be commented out